packages/@office-iss/react-native-win32/src/Libraries/Components/Text/TextWin32.Props.ts

```diff
@@ -19,6 +19,8 @@ export type TextWin32TextStyle =
   | 'HugeStandard';
 
 export interface ITextWin32Props extends RN.TextProps {
+  'aria-multiselectable'?: boolean | undefined; // Win32
```

Comment:
This should be coming in via RN.TextProps already, I think? RN.TextProps extends AccessibilityProps, so if you're adding to AccessibilityProps, you should be good, I think

Reply:
I thought I got a lint error saying it wasn't on Text props so that's why I added it. Just tested it again though and don't see it anymore so I'll update it.